README.md

```diff
@@ -33,12 +33,12 @@ The system comprises the following components:
 ## Code Prerequisites
 
 In order to build and run the code, you need the following installed:
-  - [Node](https://nodejs.org/)
+  - [Node](https://nodejs.org/) (lts/gallium - `nvm use lts/gallium`)
```

Comment:
```suggestion
  - [Node 16](https://nodejs.org/en/download/) (lts/gallium)
```

Reply:
Let's specify the number and drop nvm. I don't want people to think that's a requirement.